wsdl/ver10/schema/onvif.xsd

```diff
@@ -7759,6 +7759,25 @@ and sample rate. </xs:documentation>
 					</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:element name="SegmentDurationOverride" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Value of the active OverrideSegmentDuration parameters if one exists. (readonly)</xs:documentation>
+				</xs:annotation>
+				<xs:complexType>
+					<xs:sequence>
+						<xs:element name="Duration" type="xs:duration" minOccurs="1">
+							<xs:annotation>
+								<xs:documentation>Current value of the segment duration.</xs:documentation>
+							</xs:annotation>
+						</xs:element>
+						<xs:element name="Expiration" type="xs:duration" minOccurs="1">
+							<xs:annotation>
```

Comment:
Expiration should be an xs:dateTime. As it is defined now the value varies on each response and the termination time will jitter depending on the communication delay.

Reply:
Initially we were concerned this would jitter based on drift between caller and device times. But I don't feel super strongly on either so I'm fine with changing it if you believe this is clearer for the API

Reply:
I don't think the latency or jitter between the client and server has any critical impact in this scenario. Even if it took 5s for the device to receive the command, the device will at most do short file durations for the duration specified in the request.